### PR TITLE
Dev > Prod

### PR DESCRIPTION
**Pull request by claireriley**
TW-718 add mentions for Private API Network APIs #4518
TW-837 Custom Domain - Adding DNS records needs clarity #4521
TW 608 Team profile upgrades #4477
TW-819 flows add content structured results #4517
TW-798 Flows - edit chaining requests with data #4523
TW-815 flows edit manipulating data #4516
TW-856 Fix Postman API documentation link #4526
TW-843 flows auth tutorial #4515
flip back footer to API data, match navbar dummy data with api data #4525
TW-857 fix 404s #4528
TW-354 Microsoft AD FS SSO #4524

### CODE REVIEW

File: src/pages/docs/postman-flows/flows-intro/passing-data-between-blocks.md [hunk 1]
```diff
@@ -0,0 +1,237 @@
+---
+title: "Passing data between blocks"
+updated: 2022-01-18
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/postman-flows/flows-intro/passing-data-between-blocks.md:3]:
```suggestion
updated: 2023-01-18
```

File: src/pages/docs/publishing-your-api/custom-doc-domains.md [hunk 2]
```diff
@@ -21,69 +19,81 @@ contextual_links:
...
 
-## Adding DNS records
+### Adding DNS records
 
-For DNS records, verify the ownership with a DNS web service provider. In a separate browser tab, go to the DNS provider’s console and add the following two new records.
+To add DNS records to your domain, open another browser tab and sign in to your domain registrar or DNS provider. Add a TXT record and a CNAME record to your domain using the tokens provided by Postman.
 
-* A TXT record for `postman-echo.com` verifies the ownership of the domain. The value must be the same as the token shown in the modal. Add the TXT record to the __root domain__. For example if you're adding `example.com` or `api.example.com`, add the TXT record to `example.com`.
-* A CNAME record for `docs.postman-echo.com`, as shown in the modal. This value must be `phs.getpostman.com`. Add the CNAME record to the URL that's associated with your public documentation.
+* **TXT record** - Add a new TXT record to your domain. For **Name**, enter `@`to add the record to the root domain. For **Value**, use the TXT token string copied from Postman.
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/publishing-your-api/custom-doc-domains.md:63]:
```suggestion
* **TXT record** - Add a new TXT record to your domain. For **Name**, enter `@` to add the record to the root domain. For **Value**, use the TXT token string copied from Postman.
```

**Review by MatthewPinsker-postman — APPROVED** [review]
Looks good!

**Review by petewitcher — APPROVED** [review]
LGTM!

**Review by HeatherStenson — APPROVED** [review]
:fishsticks: 👍

**Pull request merged**